packages/npm-dependencies-validation/test/depIssuesFixer.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { expect } from "chai";
+import { resolve, join, dirname } from "path";
+import { rmdirSync, unlinkSync } from "fs";
+import { OutputChannel } from "../src/types";
+import { fixDependencyIssues, findDependencyIssues } from "../src/api";
+import { isPathExist } from "../src/utils/fileUtil";
+
+describe("`fixDependencyIssues()` function ", function () {
+  const outputChannel: OutputChannel = {
+    append: (data: string) => console.log(data),
+  };
+
+  const packageJsonPath = resolve(
+    "./test/packages-samples/positive/fix_missing_deps/package.json"
+  );
+
+  afterEach(async () => {
+    const packagePath = dirname(packageJsonPath);
+    const nodeModulesPath = join(packagePath, "node_modules");
+
+    if (await isPathExist(nodeModulesPath)) {
+      rmdirSync(nodeModulesPath, { recursive: true });
+    }
+
+    const packageLockPath = join(packagePath, "package-lock.json");
+    if (await isPathExist(packageLockPath)) {
+      unlinkSync(packageLockPath);
+    }
+  });
+
+  it("will not fix missing deps", async () => {
+    await expect(fixDependencyIssues("non_existing_package_json_path")).to.be
+      .fulfilled;
+  });
+
+  it("will fix missing deps", async function () {
+    this.timeout(20000);
```

Comment:
use `npmSpawnTestTimeout` variable in config.ts

Reply:
8000ms is not enough for this test

Reply:
okay, as we discussed we can multiply the variable by some factor